server.js

```diff
@@ -4019,6 +4019,44 @@ cache(function(data, match, sendBadge, request) {
   });
 }));
 
+// wordpress plugin active installs integration.
+// example: https://img.shields.io/wordpress/plugin/ai/akismet.svg for https://wordpress.org/plugins/akismet
+camp.route(/^\/wordpress\/plugin\/ai\/(.*)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var plugin = match[1];  // eg, `akismet`.
+  var format = match[2];
+  var apiUrl = 'http://api.wordpress.org/plugins/info/1.0/' + plugin + '.json?fields=active_installs';
+  var badgeData = getBadgeData('active installs', data);
```

Comment:
wordpress only supplies a 'active installs' value, only active plugins report to wordpress.org that they are in use on a site, so there won't be a inactive installs but i think that showing just installs could be confusing since wordpress.org uses the term 'active installs' as well